scripts/windows-setup.ps1

```diff
@@ -48,13 +48,15 @@ if (!$SkipTools) {
 
         Invoke-WebRequest -UseBasicParsing -Uri 'https://get.scoop.sh' `
             | Invoke-Expression
+        scoop install git go nvm
         scoop bucket add versions
```

Comment:
We no longer need the `versions` bucket. (Also, we can combine the two `scoop install` lines.)

Reply:
Agreed.  Removed versions and combined the scoop install lines.